crypto/marshal_test.go

```diff
@@ -0,0 +1,20 @@
+package crypto
+
+import (
+	"testing"
+)
+
+const (
+	testKeyFile = "testdata/test-keys.pem"
+)
+
+func TestParseKeyFile(t *testing.T) {
+	curves, keys, err := ParseKeyFile(testKeyFile)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	if len(keys) != 2 || len(curves) != 2 {
+		t.Fatalf("bad ParseKeyFile: curves %d, keys %d", len(curves), len(keys))
+	}
```

Comment:
maybe file name should be parse_key_test.go
should we also check what happens when a key is malformed?